config/initializers/dalli.rb

```diff
@@ -0,0 +1,23 @@
+module Memcached
+  def self.current
+    @current
+  end
+
+  def self.current=(current)
+    @current = current
+  end
+end
+
+if ENV["MEMCACHIER_SERVERS"]
+  Memcached.current = Dalli::Client.new(
+    (ENV["MEMCACHIER_SERVERS"] || "").split(","),
+    username: ENV["MEMCACHIER_USERNAME"],
+    password: ENV["MEMCACHIER_PASSWORD"],
+    failover: true,
+    socket_timeout: 1.5,
+    socket_failure_delay: 0.2,
+    expires_in: 15 * 60
```

Comment:
Remove this, per https://github.com/bundler/bundler-api/commit/dca01425b01c806a7a4d43b6bb188d2666651f9f